tycho-plugins/target-platform-utils/src/main/java/org/jboss/tools/tycho/targets/TargetToRepoMojo.java

```diff
@@ -72,14 +78,17 @@
 	@Component
     private RepositorySystem repositorySystem;
 
-	@Parameter
+	@Parameter(property = "targetDefinition")
     private File sourceTargetFile;
 
 	@Parameter
 	private TargetArtifact sourceTargetArtifact;
 
-	@Parameter(property = "mirror-target-to-repo.includeSources")
+	@Parameter(property = "mirror-target-to-repo.includeSources", defaultValue = "false")
```

Comment:
we set mirror-target-to-repo.includeSources=true for all the JBT/devstudio TPs -- so we COULD turn this to defaultValue = true if we want... not sure which is the best default. Don't most devs WANT sources by default?

Reply:
Right, but changing the default value constitute a relatively "breaking" changes for adopters. So it's IMO safer to keep default behavior the same.

Reply:
OK sounds good.